lib/openstudio-standards/standards/Standards.Model.rb

```diff
@@ -4653,45 +4676,65 @@ def model_apply_prm_baseline_window_to_wall_ratio(model, climate_zone, wwr_build
           when 'Unconditioned'
             next # Skip unconditioned spaces
           when 'NonResConditioned'
-            next unless vals['red_nr']
-
             mult = vals['mult_nr_red']
+            total_wall_area = vals['nr_wall_m2']
+            total_wall_with_fene = vals['nr_fene_only_wall_m2']
```

Comment:
Thanks for the suggestion. I agree the `total_wall_area` should rename to avoid confusion. However, the `total_wall_with_fene` is not exactly the `total_net_wall_area`

`total_wall_with_fene` is the total area of the vertical surfaces that have fenestrations. It is still the gross area of walls. The difference is that the wall gross area is only added if the wall has fenestration surfaces. I am using this parameter to determine what is the maximum WWR a model can reach by only expanding the existing windows.

Reply:
I see, that makes sense! Would it be alright just adding the `_area` suffix to be consistent with the other variables?